### PR TITLE
Rebalances the loadout, re-adds tablets and adds laptops to loadout

### PR DESCRIPTION
**Pull request by MarinaGryphon**
* Adjusts a lot of loadout items to have different costs depending on whether they're job-specific, storage items, utility items, etc.
* Reduces the default loadout item cost to 1/2, however many items have had their costs increased either back to normal or higher to compensate. Examples of more expensive items: things that replace uniforms, non-job-restricted items, items with functions that couldn't be obtained as easily by other means.
* Re-adds tablets to the loadout.
* Adds laptops to the loadout.
* Unifies the various flasks in loadout, allows you to specify temperature for them and lets you choose alcoholic OR non-alcoholic beverages.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
...are these fractions?

**Comment by MarinaGryphon** [comment]
> ...are these fractions?

yes.

**Comment by alsoandanswer** [comment]
My brain is too small for these calculations

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -86,16 +86,19 @@
 	display_name = "webbing, engineering"
 	path = /obj/item/clothing/accessory/storage/brown_vest
 	allowed_roles = list("Station Engineer", "Atmospheric Technician", "Chief Engineer", "Engineering Apprentice")
+	cost = 3/2
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm]: Not a fan of fractions or even decimals as costs.

**Comment by NonQueueingMatt** [comment]
Inactive, merge conflicts. Reopen when ready.

**Pull request closed without merging**